### DISCUSSION TITLE
Initial feedback on V2 portal and API

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/orcahello#198] [Ideas]
Add your quick thoughts here about the amazing new "version 2" of the OrcaHello portal that Mike @micowan has pioneered. Thanks, Mike, and thank you to all the 2024 Hackathon participants who helped get v2 beta launched and who provide feedback here.

As discussion proceeds, we can distill key observations and the highest-priority ideas into new Issues.

- [New (v2) OrcaHello Portal](https://aifororcas2.azurewebsites.net/)
- [New (v2) OrcaHello API](https://aifororcasdetections2.azurewebsites.net/swagger/index.html)

For reference, as of 9/19/2024 here are the v1 links:

- [Old (v1) OrcaHello Portal](https://aifororcas.azurewebsites.net/)
- [Old (v1) OrcaHello API](https://aifororcasdetections.azurewebsites.net/swagger/index.html)

**Comment by scottveirs** [comment]
A few thoughts regarding the `Orca Sounds` page at [https://aifororcas2.azurewebsites.net/orca_sounds](https://aifororcas2.azurewebsites.net/orca_sounds) --

1. When I first selected the `Orca Sounds` link from the left menu, I anticipated a gallery of common sounds made by orcas (like what we present to train human listeners here and here). I had seen the `Explore` menu item and imagined that's where a subscriber (not logged in) would go to filter through AI detections... An alternative menu label and page title for the `Orca Sounds` content that would have been more accurate (less confusing) for me would be `AI reports` or `Detected orca calls`
2. For logged in users (i.e. moderators ready to work), the default view of `Unreviewed Calls` makes good sense (although we could adopt the nomenclature we're using for human detections grouped in time for moderation ("candidates") and instead use `Unreviewed candidates.` BUT for subscribers (users who don't log in) I think it would be cool if the default view was instead `Verified` candidates with a default time period of the last month or two and a limit of 5 or 10 records. That way public users would get to listen to orca calls in true positive candidates, rather than seeing a Unreviewed candidate queue -- one that ideally/usually will be empty if the moderators are efficient at processing them.
3. I noticed a minor typo (`Verfied` instead of `Verified`) in the dropdown menu for Unreviewed/Verified/etc and so created this quick-fix issue (#199).

**Comment by micowan** [comment]
Scott,

Overall, I think these a good recommendations. I would recommend you reach out to Claire to get them rolled into the Dashboard changes she is proposing.


Michael Cowan | Senior Consultant

Microsoft Corporation

________________________________
